<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="CHI-CL-6RJGL13" name="Karate.KarateScripts.APIRunnerTest" tests="1" failures="1" timestamp="2021-09-13T21:33:04 IST" time="5.547" errors="0">
  <testcase name="parallelRun" time="5.547" classname="Karate.KarateScripts.APIRunnerTest">
    <failure type="java.lang.AssertionError" message="failed features:
resources.TwoWAyCommands: -unknown-:40 - 
TokenTwoWay.feature:21 - status code was: 429, expected: 200, response time: 1218, url: https://restdev1.tracking.thermoking.com/connected-solutions/obtainToken/, response: {&quot;path&quot;:&quot;/connected-solutions/obtainToken/&quot;,&quot;error&quot;:&quot;Too Many Requests&quot;,&quot;message&quot;:&quot;This client has been exceeded the allowance of requests for this API&quot;,&quot;timestamp&quot;:&quot;2021-09-13T16:03:01.686+0000&quot;,&quot;status&quot;:429}
">
      <![CDATA[java.lang.AssertionError: failed features:
resources.TwoWAyCommands: -unknown-:40 - 
TokenTwoWay.feature:21 - status code was: 429, expected: 200, response time: 1218, url: https://restdev1.tracking.thermoking.com/connected-solutions/obtainToken/, response: {"path":"/connected-solutions/obtainToken/","error":"Too Many Requests","message":"This client has been exceeded the allowance of requests for this API","timestamp":"2021-09-13T16:03:01.686+0000","status":429}

at org.junit.Assert.fail(Assert.java:88)
at org.junit.Assert.assertTrue(Assert.java:41)
at Karate.KarateScripts.APIRunnerTest.parallelRun(APIRunnerTest.java:54)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:136)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:145)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
    </failure>
  </testcase> <!-- parallelRun -->
  <system-out/>
</testsuite> <!-- Karate.KarateScripts.APIRunnerTest -->
